src/nextjs/templates.ts

```diff
@@ -216,22 +216,15 @@ export default function Page() {
             margin: "18px",
           }}
           onClick={async () => {
-            const transaction = Sentry.startTransaction({
-              name: "Example Frontend Transaction",
-            });
-
-            Sentry.configureScope((scope) => {
-              scope.setSpan(transaction);
-            });
-
-            try {
+            await Sentry.startSpan({
+              name: 'Example Frontend Span',
+              op: 'test'
+            }, async () => {
```

Comment:
We could remove the async on the onClick handler and also the await before Sentry.startSpan

Reply:
good point, removed it in both templates